### PR TITLE
Fix issues with user list, passwords and app reloading

### PR DESCRIPTION
**Pull request by thivi**

### CODE REVIEW

File: apps/console/src/features/authentication/pages/sign-in.tsx [hunk 1]
```diff
@@ -44,14 +46,14 @@ const SignIn: FunctionComponent<RouteComponentProps> = (
     const loginSuccessRedirect = () => {
 
         // TODO: Fix the auth callback isssue here.
```
Comment (brionmario) [inline apps/console/src/features/authentication/pages/sign-in.tsx:48]: If this fixes the issue, shall we remove the TODO.

File: modules/i18n/src/translations/si-LK/portals/common.ts [hunk 2]
```diff
@@ -106,29 +109,32 @@ export const common: CommonNS = {
...
+    tooShort: "බොහෝ කෙටි",
     type: "වර්ගය",
     unpin: "ඉවත් කරන්න",
     unpinned: "ඉවත් කරන ලදි",
     update: "යාවත්කාලීන කරන්න",
     user: "පරිශීලක",
     verify: "සත්‍යාපනය කරන්න",
-    view: "බලන්න"
+    view: "බලන්න",
+    weak: "දුර්වල",
```
Comment (brionmario) [inline modules/i18n/src/translations/si-LK/portals/common.ts:138]:
```suggestion
    weak: "දුර්වලයි",
```

File: apps/myaccount/src/components/authentication/sign-in.tsx [hunk 3]
```diff
@@ -44,14 +46,15 @@ const SignIn: FunctionComponent<RouteComponentProps> = (
     const loginSuccessRedirect = () => {
 
         // TODO: Fix the auth callback isssue here.
-        /* const AuthenticationCallbackUrl = AuthenticateUtils.getAuthenticationCallbackUrl();
+        const AuthenticationCallbackUrl = AuthenticateUtils.getAuthenticationCallbackUrl(
```
Comment (brionmario) [inline apps/myaccount/src/components/authentication/sign-in.tsx:49]: Have you tested the flow in tenant mode?

If it works fine in tenant mode as well, we can remove the `TODO` comment.

**Review by brionmario — APPROVED** [review]

File: apps/myaccount/src/components/authentication/sign-in.tsx (hunk 3 above)
Comment (thivi) [reply apps/myaccount/src/components/authentication/sign-in.tsx:49]: I will test it in the tenant mode and see if this works fine.

File: apps/console/src/features/authentication/pages/sign-in.tsx (hunk 1 above)
Comment (thivi) [reply apps/console/src/features/authentication/pages/sign-in.tsx]: Resolved by 32527d6

File: apps/myaccount/src/components/authentication/sign-in.tsx (hunk 3 above)
Comment (thivi) [reply apps/myaccount/src/components/authentication/sign-in.tsx:48]: I tested it and it works fine.

**Review by brionmario — APPROVED** [review]

**Pull request merged**